src/VCAuthn/Controllers/PresentationConfigurationController.cs

```diff
@@ -0,0 +1,57 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using VCAuthn.PresentationConfiguration;
+
+namespace VCAuthn.Controllers
+{
+    [Route("api/vc-configs")]
+    [ApiController]
+    public class PresentationConfigurationController : ControllerBase
+    {
+        private readonly IPresentationConfigurationService _service;
+
+        public PresentationConfigurationController(IPresentationConfigurationService service)
+        {
+            _service = service;
+        }
+        
+
+        // GET: api/vc-configs/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<PresentationRecord>> GetConfig(string id)
+        {
+            var record = await _service.GetAsync(id);
+
+            if (record == null)
+            {
+                return NotFound();
+            }
+
+            return Ok(record);
+        }
+        
+        // POST: api/vc-configs
+        [HttpPost]
+        public async Task<ActionResult> CreateConfig([FromBody] PresentationRecord record)
+        {
+            await _service.CreateAsync(record);
+            return Ok();
+        }
+        
+        // PUT: api/vc-configs
+        [HttpPut]
+        public async Task<ActionResult> UpdateConfig([FromBody] PresentationRecord record)
+        {
+            await _service.UpdateAsync(record);
+            return Ok();
+        }
+        
+        // DELETE: api/vc-configs/5
+        [HttpDelete("{id}")]
+        public async Task<ActionResult> DeleteConfig(string id)
+        {
+            await _service.DeleteAsync(id);
+            return Ok();
```

Comment:
Should this be a `204 No Content`?

Reply:
Don't see a difference in this case, to be honest. Would it be mainly to broaden the response palette? :)